testem.safari.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+/* eslint-env node */
+module.exports = {
+  test_page: 'tests/index.html?hidepassed',
+  disable_watching: true,
+  parallel: 1,
```

Comment:
Do I assume correctly that we need a separate `testem.safari.js` and cannot include it into the `testem.js` because we need to set `parallel = 1` only for Safari?

Reply:
I wanted the Safari runner only to run in the MacOS configuration. Placing it in the normal `testem` file would probably cause it to attempt to run on the Linux CI actions as well. 
This setup should be similar to how BrowserStack worked in the past.

There could be a way to have this all in the same file, but this is just the one I stumbled on that worked :sweat_smile: